build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs

```diff
@@ -0,0 +1,59 @@
+# *REQUIRED*
```

Comment:
Just curious to know from where you got this file or its contents. The rest of fedora config files here I have the feeling that with some patience and trials I could have produced them, but I am not that sure about this one. It is already provided by some package and just slightly patched?

Reply:
I don't know either :) I copied those from a fedora description of the past

Reply:
maybe Neal has an idea if we still need it

@Conan-Kudo thoughts ?

Thanks

Reply:
Why do we need any of these? Shouldn't this be provided by Fedora packages already?

Reply:
To be honest I blindly copied the files as they were present on the other fedora images we have in the kiwi-descriptions repo. I could imagine that these files are provided by fedora packages but maybe the config is adapted for the needs of an image. I'm happy to reduce the number of overlay files, really

Reply:
So you are saying we better drop it ?

Reply:
I dropped the file and will merge the test descriptions now. As they do not server a customer purpose we can adapt when needed.